### PR TITLE
standards, styles, edits for Tutorial 2: Private Inputs and Hash Functions

### PR DESCRIPTION
**Pull request by barriebyron**
Tutorial 2: Private Inputs and Hash Functions 
closes https://app.zenhub.com/workspaces/zkapps-product-eng-6130fedb3b0fc600123d8796/issues/gh/o1-labs/docs2/297
This PR:
- provided guided steps to copy the entire example files like tuto 1
- applies styles and standards
- changes pronouns per style
- adds a description of cryptographic salt
- updates versions Mina zkApp CLI 0.9.0` and SnarkyJS 0.11.0

### CODE REVIEW

File: docs/zkapps/tutorials/01-hello-world.mdx [hunk 1]
```diff
@@ -222,9 +222,6 @@ All functions used inside a smart contract must operate on SnarkyJS compatible d
 
 :::info
 
-
```
Comment (barriebyron) [inline docs/zkapps/tutorials/01-hello-world.mdx:225]: not related to tuto2 but redundant so did a little edit dance here too

**Review by MartinMinkov — APPROVED** [review]

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx [hunk 2]
```diff
@@ -24,99 +24,194 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 :::
 
-:::note
 
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.8.0.
+# Tutorial 2: Private Inputs and Hash Functions
 
-:::
+In the [Hello World](hello-world) tutorial, you built a basic zkApp smart contract in SnarkyJS with a single state variable that could be updated if you knew the square of that number.
```
Comment (ymekuria) [inline docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx:30]: nit: 
built a basic zkApp smart contract `in` SnarkyJS -> built a basic zkApp smart contract `with` SnarkyJS

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx [hunk 3]
```diff
@@ -24,99 +24,194 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
...
 ```
 
-This just adds the basic setup for our smart contract — see [Tutorial 01 Hello World](hello-world) for more on this.
+### Exports
 
-Now, we will add an `initState()` method. This is intended to run once to set up the initial state on the zkApp account.
+The smart contract called `IncrementSecret` has one element of on-chain state named `num` of type `Field` as defined by following code:
 
-```ts
+```ts ignore
```
Comment (ymekuria) [inline docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx:185]: What does adding ignore do here?

**Review by ymekuria — APPROVED** [review]
This looks great @barriebyron! I just had a minor comment and question.

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx (hunk 2 above)
Comment (barriebyron) [inline docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx:30]:
```suggestion
In the [Hello World](hello-world) tutorial, you built a basic zkApp smart contract with SnarkyJS with a single state variable that could be updated if you knew the square of that number.
```

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx (hunk 3 above)
Comment (ymekuria) [reply docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx:185]: Thanks for the explanation @barriebyron !

**Pull request merged**